packages/gatsby/src/utils/merge-gatsby-config.js

```diff
@@ -51,7 +51,10 @@ const howToMerge = {
   // plugins are concatenated and uniq'd, so we don't get two of the same plugin value
   plugins: (a = [], b = []) =>
     _.uniqWith(a.concat(b), (a, b) =>
-      _.isEqual(normalizePluginEntry(a), normalizePluginEntry(b))
+      _.isEqual(
+        _.omit(normalizePluginEntry(a), [`parentDir`]),
+        _.omit(normalizePluginEntry(b), [`parentDir`])
```

Comment:
maybe we should actually use `_.pick(normalizePluginEntry(x), [`resolve`, `options`])` instead of `_.omit`? Doesn't impact current code but well safe-guard against any future additions?

Reply:
I agree, that would be better.  I don't usually use lodash for very much, so I wasn't aware of the `pick` function, but it makes much more sense.  I've updated it